src/queries/lunch.js

```diff
@@ -0,0 +1,44 @@
+import gql from 'graphql-tag';
+
+export const GET_LUNCHES = gql`
+  query Lunches($ids: [ID], $limit: Int, $offset: Int, $order: LunchesOrder, $dateRange: [String]) {
+    lunches(ids: $ids, limit: $limit, skip: $offset, order: $order, dateRange: $dateRange) {
+      id
+      text
+      lunchOffers {
+        name
+        price
+      }
+      pointOfInterest {
+        id
+        addresses {
+          kind
+          city
+          street
+          zip
+        }
+        contact {
+          email
+          fax
+          phone
+          webUrls {
+            id
+            url
```

Comment:
i miss `description` here

Reply:
added it to the query. the types in LunchSection already accomodated it, i just missed it in the query...